src/components/MAPI/organizations/AccessControl/utils.ts

```diff
@@ -550,16 +543,11 @@ export async function ensureServiceAccount(
   auth: IOAuth2Provider,
   name: string,
   namespace: string
-): Promise<corev1.IServiceAccount> {
+): Promise<ui.IAccessControlServiceAccounts> {
```

Comment:
Changed this to return an object that associates the service account name with its status.